src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx

```diff
@@ -6,6 +6,7 @@ import { TestRunTestWithRecordings } from "shared/test-suites/TestRun";
 import { useTestRunDetailsSuspends } from "../TestRuns/hooks/useTestRunDetailsSuspends";
 import { TestResultListItem } from "./Overview/TestResultListItem";
 import { TestRunsContext } from "./TestRunsContextRoot";
+import styles from "../../../Testsuites.module.css";
```

Comment:
Same comment as this: https://github.com/replayio/devtools/pull/9979/files#r1418926102